guides/common/modules/con_how-puppet-integrates-with-project.adoc

```diff
@@ -45,7 +45,7 @@ endif::[]
 ifndef::satellite[]
 by syncing repositories in custom products
 endif::[]
-and by using {ContentManagementDocURL}Managing_Activation_Keys_content-management[Activation Keys] and {ContentManagementDocURL}Managing_Content_Views_content-management[Content Views].
+and by using {ContentManagementDocURL}Managing_Activation_Keys_content-management[Activation Keys] and {ContentManagementDocURL}Managing_Content_Views_content-management[content views].
```

Comment:
```suggestion
and by using {ContentManagementDocURL}Managing_Activation_Keys_content-management[activation keys] and {ContentManagementDocURL}Managing_Content_Views_content-management[content views].
```
Unrelated, but while you're at it might as well.

Reply:
+1